Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -1138,6 +1138,18 @@ - (void)textViewDidEndEditing:(UITextView *)textView
     [textView resignFirstResponder];
 }
 
+- (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text
```

Comment:
@michaelkirk This is a significant change so that users using bluetooth keyboards can send messages with the return key.  

You can still paste in multi-line messages, but you can't type them anymore.

I've changed the keyboard's return button to read send.